.vscode/settings.json

```diff
@@ -13,8 +13,12 @@
 			}
 	],
 
+	"vim.hlsearch": false,
```

Comment:
TBH, I feel like we shouldn't be checking in these settings as they are very much user-specific.